lte/gateway/configs/pipelined.yml

```diff
@@ -36,6 +36,7 @@ static_services: [
   'access_control',
   'ryu_rest_service',
   'startup_flows',
+  'local_node_state',
```

Comment:
@ulaskozat should we create a new top level module in magma for the 5GC application stuff? I imagine we wouldn't need to run this pipelined app on the LTE AGW, for example right? Also there's a hardcoded reference to an `smfd` service later in this PR which is also something we probably wouldn't run on AGW I'm assuming.

This would let us separate the gateway configs between 5GC and LTE, create new 5GC-only mconfigs, and so on. Thoughts?

Reply:
@xjtian the `smfd` thing will be sessiond service itself. The goal is to have a converged session management and converged user plane functionality. Having said that indeed we should not make any 5G service as a mandatory app.